### ISSUE TITLE
Need way to disable SSL checks

### ISSUE THREAD

**Issue by dcapwell** [tedops/party#10]
`requests` allows me to disable SSL verification, but don't see how to add this to party; it would be nice if I could add this to the config.

```
  File "/usr/lib64/python2.7/site-packages/party/party.py", line 99, in find
    raw_response = self.query_artifactory(query)
  File "/usr/lib64/python2.7/site-packages/party/party.py", line 37, in query_artifactory
    response = requests.get(query, auth=auth, headers=self.headers)
  File "/usr/lib/python2.7/site-packages/requests/api.py", line 70, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/lib/python2.7/site-packages/requests/api.py", line 56, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python2.7/site-packages/requests/sessions.py", line 488, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/site-packages/requests/sessions.py", line 609, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/site-packages/requests/adapters.py", line 497, in send
    raise SSLError(e, request=request)
SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:579)
```

**Comment by sijis** [comment]
You should be able to add, temporarily, the changes to the `query_artifactory()` method in your installed version of party.

Im thinking a parameter/option in party_config.py file would work? It would default to True but can be changed via a config globally. Would that work?

**Comment by tedops** [comment]
@dcapwell @sijis This has been implemented in v1.7.2 via an added parameter in party_config.py. Please try out the new version and let me know if that works for you. Thanks!
